build/variables.yml

```diff
@@ -62,25 +65,25 @@ variables:
   # AzureStorageAccountName:
 
   # Solution to build
-  SolutionFileName: ApplicationTemplate.sln
+  SolutionName: ApplicationTemplate
 
   # Paths to file
-  InfoPlistPath: $(Build.SourcesDirectory)/src/app/ApplicationTemplate.iOS/Info.plist
-  AndroidManifestPath: $(Build.SourcesDirectory)/src/app/ApplicationTemplate.Droid/Properties/AndroidManifest.xml
+  InfoPlistPath: '$(Build.SourcesDirectory)/src/app/$(SolutionName).Mobile/iOS/Info.plist'
+  AndroidManifestPath: '$(Build.SourcesDirectory)/src/app/$(SolutionName).Mobile/Android/AndroidManifest.xml'
 
-  # Pool names
-  windowsPoolName: 'windows 2022'
+  # Virtual machine images
+  windowsHostedAgentImage: 'windows-2022'
+  macOSHostedAgentImage: 'macOS-12'
   macOSPoolName: 'macOS'
```

Comment:
@jeanplevesque Should we use hosted machine for all the pipelines?

Reply:
We can review and adjust based on performance. You can add an committee item to measure and adjust.

Reply:
Tried on our agent never worked.